### PR TITLE
fix(@angular/cli): ng get: return whole config root when no path provided.

### LINKED ISSUE #5886: ng get causes InvalidJsonPath error
### Versions.
ps>ng --version
    _                      _                 ____ _     ___
   / \   _ __   __ _ _   _| | __ _ _ __     / ___| |   |_ _|
  / △ \ | '_ \ / _` | | | | |/ _` | '__|   | |   | |    | |
 / ___ \| | | | (_| | |_| | | (_| | |      | |___| |___ | |
/_/   \_\_| |_|\__, |\__,_|_|\__,_|_|       \____|_____|___|
               |___/
@angular/cli: 1.0.0
node: 7.7.2
os: win32 x64
@angular/common: 4.0.1
@angular/compiler: 4.0.1
@angular/core: 4.0.1
@angular/forms: 4.0.1
@angular/http: 4.0.1
@angular/platform-browser: 4.0.1
@angular/platform-browser-dynamic: 4.0.1
@angular/router: 4.0.1
@angular/cli: 1.0.0
@angular/compiler-cli: 4.0.1

### Repro steps.

1. Navigate to an existing directory that was created via ng new, such as ng new angular-hello.
2. Open a prompt at the location of the project directory.   Example c:\projects\angular-hello
3. ng get

### The log given by the failure.
ps>ng get --verbose 
The option '--verbose' is not registered with the get command. Run `ng get --help` for a list of supported options.
InvalidJsonPath
Error: InvalidJsonPath
    at _parseJsonPath (C:\projects\angular-hello\node_modules\@ngtools\json-schema\src\schema-class-factory.js:29:19)
    at _getSchemaNodeForPath (C:\projects\angular-hello\node_modules\@ngtools\json-schema\src\schema-class-factory.js:41
:21)
    at GeneratedSchemaClass.$$get (C:\projects\angular-hello\node_modules\@ngtools\json-schema\src\schema-class-factory.
js:84:22)
    at CliConfig.get (C:\projects\angular-hello\node_modules\@angular\cli\models\config\config.js:31:29)
    at resolve (C:\projects\angular-hello\node_modules\@angular\cli\commands\get.js:25:34)
    at Class.run (C:\projects\angular-hello\node_modules\@angular\cli\commands\get.js:19:16)
    at Class.<anonymous> (C:\projects\angular-hello\node_modules\@angular\cli\ember-cli\lib\models\command.js:134:17)
    at process._tickCallback (internal/process/next_tick.js:109:7)

### Desired functionality.
Eliminate the InvalidJsonPath error

### Mention any other details that might be useful.

1. ng build of the project works.
2. ng serve of the project works.

### PR DESCRIPTION
**Pull request by prestonvanloon**
Running `ng get` without a config key would throw a `invalidJsonPath` error rather than a meaningful message. 

This PR
- returns the whole config object when no json path provided.
- pretty prints the object using `JSON.stringify(value, null, 2)`.

Fixes #5886

### CODE REVIEW

File: packages/@angular/cli/commands/get.ts [hunk 1]
```diff
@@ -11,7 +11,7 @@ export interface GetOptions {
 
 const GetCommand = Command.extend({
   name: 'get',
-  description: 'Get a value from the configuration.',
+  description: 'Get a value from the configuration. Example: ng get project.name',
```
Comment (sumitarora) [inline packages/@angular/cli/commands/get.ts]: @prestonvanloon Please change this to match format `eg. "ng new [name]".`

File: tests/e2e/tests/commands/get/get.ts [hunk 2]
```diff
@@ -4,6 +4,7 @@ import {expectToFail} from '../../../utils/utils';
 export default function() {
   return Promise.resolve()
     .then(() => expectToFail(() => ng('get', 'apps.zzz.prefix')))
+    .then(() => expectToFail(() => ng('get', undefined)))
```
Comment (sumitarora) [inline tests/e2e/tests/commands/get/get.ts]: This should be just `ng('get')` as `ng get undefined` seems to be working fine already as it is converted to string key.

File: packages/@angular/cli/commands/get.ts (hunk 1 above)
Comment (prestonvanloon) [inline packages/@angular/cli/commands/get.ts]: Done.

File: tests/e2e/tests/commands/get/get.ts (hunk 2 above)
Comment (prestonvanloon) [inline tests/e2e/tests/commands/get/get.ts]: Done.

**Comment by prestonvanloon** [comment]
I think this should be closed in favor of returning the whole config object when no param specified. 

I'll submit a new PR later.

**Comment by prestonvanloon** [comment]
Or this PR modified to return the whole object. Not sure what is preferred.

**Comment by prestonvanloon** [comment]
I updated this PR title, description, and content with the changes outlined in my previous comment.

**Comment by filipesilva** [comment]
Heya @prestonvanloon thanks for the work here. Can you rebase this PR? There's a couple of CI errors that were due to changes on our side, but it should be fine now.

**Comment by prestonvanloon** [comment]
@filipesilva OK. Should be all good now. Thanks!

**Pull request closed without merging**